packages/control-property-editor-common/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+    "extends": ["../../.eslintrc"],
+    "parserOptions": {
+        "EXPERIMENTAL_useSourceOfProjectReferenceRedirect": true,
+        "project": "./tsconfig-test.json"
+    },
+    "overrides": [
+        {
+            "parser": "@typescript-eslint/parser",
+            "files": ["./test/**/*.ts"],
+            "rules": {
+                "no-loop-func": "off"
+            }
+        }
+    ],
+    "rules": {
+        "no-console": "off"
```

Comment:
Why it be ok for you to use console statements?

Reply:
removed

Reply:
Same in the `control-property-editor` please

Reply:
I have tried to use  Toolslogger in CPE but the app fails to load because its trying to run node js code
```
export const logger = new ToolsLogger({
    transports: [new ConsoleTransport()],
    logPrefix: 'Control-Property-Editor'
});
```

Reply:
You cannot use the tools logger in the browser. If you are in the UI5 world, use `sap/base/Log` and if you are in the react world use whatever is valid there.

Reply:
I am good with the changes you did. Choosing a better lib in react, I leave to the react experts